softwerkskammer/testIntegration/app_test.js

```diff
@@ -2,7 +2,7 @@
 var expect = require('must-dist');
 var httpRequest = require('request');
 var sinon = require('sinon').sandbox.create();
-var conf = require('../testutil/configureForTest');
+var conf = require('../testutil/configureForTestWithDB.js');
```

Comment:
Wegen dem global state in simple-require: Sobald ein einziger Test in einem Node-Prozess das `configureForTest` module required, bekommen alle Tests diese Beans, auch wenn sie das  `configureTestWithDB` requiren...